app/src/main/java/com/smithnoff/mytaskyapp/data/remote/TaskyApi.kt

```diff
@@ -0,0 +1,14 @@
+package com.smithnoff.mytaskyapp.data.remote
+
+import com.smithnoff.mytaskyapp.utils.Constants.LOGIN
+import com.smithnoff.mytaskyapp.utils.Constants.REGISTER
+import retrofit2.http.POST
+
+interface TaskyApi {
+
+    @POST(REGISTER)
```

Comment:
This is going to be used only once

So no need to have a constant for the endpoints, unless you are going to use them in many places

Reply:
Or if it adds readability for a magic value which it also doesn't here, it's pretty clear what this string would stand for 👍